rfcs/010-react-intl.md

```diff
@@ -0,0 +1,53 @@
+# RFC 010 - Best Practices with react-intl
+
+## Affected projects
+
+- Frontend
+
+## Motivation
+
+We're using react-intl and have currently 2 ways of expressing a message ("defineMessages + formatMessage", or "FormattedMessage"). It's currently not clear when to use one or another approach and guidelines would be useful for current and new developers.
+
+## Comparaison of the 2 approaches
+
+### defineMessages + formatMessage
+
+Pros:
+- Without JSX "bloat", concise
+- Can be assigned to a variable
+- Can be post manipulated in Javascript
+- Support values
+
+Cons:
+- 2 steps: Needs first to defineMessages then formatMessage
+
+### FormattedMessage
+
+Pros:
+- Self contained
+- Can see the defaultValue in its context
```

Comment:
I may have underestimated this one in the past, but recently realized that it's actually pretty important.